libs/event/api/src/lib/event.model.ts

```diff
@@ -13,12 +15,13 @@ import {Image} from '@wepublish/image/api'
 import {GraphQLRichText} from '@wepublish/richtext/api'
 import {PaginatedType, SortOrder} from '@wepublish/utils/api'
 import {Node} from 'slate'
+import {Page} from './page.model'
 
 export enum EventSort {
-  CreatedAt = 'CreatedAt',
-  ModifiedAt = 'ModifiedAt',
-  StartsAt = 'StartsAt',
-  EndsAt = 'EndsAt'
+  CREATED_AT = 'CREATED_AT',
```

Comment:
Why? :(

Reply:
There are differences between versions.
This is only thing left for me now in this pull-request.

E.g.

```
export enum SubscriptionDeactivationReason {
  InvoiceNotPaid = 'INVOICE_NOT_PAID',
  None = 'NONE',
  UserSelfDeactivated = 'USER_SELF_DEACTIVATED',
  InvoiceNotPaid = 'invoiceNotPaid',
  None = 'none',
  UserSelfDeactivated = 'userSelfDeactivated'
}
```